deployment/webhook-server/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1alpha1
```

Comment:
it's better to merge this chart into the default one.

Reply:
addressed